Readme.md

```diff
@@ -41,6 +41,8 @@ You can then easily install all of the dependencies with one of the following co
 
     $ brew bundle
 
+In case dependency already installed and there is update available it will be upgraded.
```

Comment:
Updated in https://github.com/timurvafin/homebrew-bundle/commit/6a911be2e911cfa4bacadf6db8fcc5289f3f3db8